add field for virtual network gateway and virtual network gateway connection (#9330)
terraform-providers/terraform-provider-azurerm: website/docs/d/virtual_network_gateway.html.markdown

* `active_active` - Is this an Active-Active Gateway?

> * `private_ip_address_enabled` - Whether private IP needs to be enabled on this gateway for connections or not.

Reviewer: grammer?
```suggestion
* `private_ip_address_enabled` - Whether a private IP will be used for this  gateway for connections.
```

or if the description is right
```suggestion
* `private_ip_address_required` - Whether private IP needs to be enabled on this gateway for connections or not.
```
